tempodb/encoding/vparquet3/schema.go

```diff
@@ -150,16 +150,27 @@ type DedicatedAttributes struct {
 	String10 *string `parquet:",snappy,optional,dict"`
 }
 
-type EventAttribute struct {
-	Key   string `parquet:",snappy,dict"`
-	Value []byte `parquet:",snappy"` // Was json-encoded data, is now proto encoded data
+type Event struct {
+	TimeSinceStartUnixNano uint64      `parquet:",delta"`
+	Name                   string      `parquet:",snappy"`
```

Comment:
Event names have probably a cardinality similar to span names. Therefore it can make sense to use dictionary encoding. 
```suggestion
	Name                   string      `parquet:",snappy,dict"`
```